### PR TITLE
Fixed issue of attribute value with pixel

### PR DESCRIPTION
**Pull request by atiqsayyed**

### CODE REVIEW

**Comment by hrj** [comment]
This looks great, thanks!

File: misc/runTest.sh [hunk 1]
```diff
@@ -41,4 +41,4 @@ python -m SimpleHTTPServer 8000 &> /dev/null &
 
 java -jar grinder.jar prepare
 java -jar grinder.jar compare gngr $GRINDER_KEY --baseLine=$HOME/grinderBaselines/gngr --uploadImg=y
-java -jar grinder.jar checkBase data ../grinderBaselines/gngr
+java -jar grinder.jar checkBase data ../grinderBaselines/gngr
```
Comment (hrj) [inline misc/runTest.sh]: For next time, avoid unrelated changes in a commit / PR.

File: src/HTML_Renderer/org/lobobrowser/html/style/HtmlValues.java [hunk 2]
```diff
@@ -579,7 +579,11 @@ public static int getOldSyntaxPixelSize(String spec, final int availSize, final
     try {
       if (spec.endsWith("%")) {
         return (availSize * Integer.parseInt(spec.substring(0, spec.length() - 1))) / 100;
-      } else {
+      }
+      if (spec.endsWith("px")){
+          return (int) Math.round(Double.parseDouble(spec.substring(0, spec.length() - 2)));
+      }
+      else {
```
Comment (hrj) [inline src/HTML_Renderer/org/lobobrowser/html/style/HtmlValues.java]: Minor comment: the coding style we are following in this project is to begin the `else` on the same line as `}`.

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 .metadata
 *.swp
+*.iml
```
Comment (hrj) [inline .gitignore]: Are you using IntelliJ successfully with this project? That's neat!

Can you try to import all Eclipse formatting settings into IntelliJ. They are located in the `misc` folder in this repo.

Comment (atiqsayyed) [inline .gitignore]: I can try to import all the formatting settings into Intellij. . Thanks
